commits d6746b1 and c711dc3 missed updating mumps

Hi,
I believe that https://github.com/NGSolve/ngsolve/commit/d6746b16e52ae18cf721959edd8a2eaad8dc4d14 and https://github.com/NGSolve/ngsolve/commit/c711dc38db58f548672dbc6f3693ad1821907441 missed updating (at least) the file `linalg/mumpsinverse.cpp`. On my FEM on Colab workflow I get the following errors
```
2023-03-01T22:16:09.1916624Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:987:18:   required from here
2023-03-01T22:16:09.1917836Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:70:35: error: 'WIDTH' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.1918791Z    70 |         if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
2023-03-01T22:16:09.1919246Z       |                                   ^~~~~
2023-03-01T22:16:09.1979692Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:70:65: error: 'HEIGHT' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.1980611Z    70 |         if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
2023-03-01T22:16:09.1981145Z       |                                                                 ^~~~~~
2023-03-01T22:16:09.2053841Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:78:33: error: 'HEIGHT' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.2054642Z    78 |     entrysize = mat_traits<TM>::HEIGHT;
2023-03-01T22:16:09.2055122Z       |                                 ^~~~~~
2023-03-01T22:16:09.2055897Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:79:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.2056513Z    79 |     iscomplex = mat_traits<TM>::IS_COMPLEX;
2023-03-01T22:16:09.2056930Z       |                                 ^~~~~~~~~~
2023-03-01T22:16:09.2295281Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::MumpsInverse<TM, TV_ROW, TV_COL>::MumpsInverse(const ngla::SparseMatrix<TM, TV_ROW, TV_COL>&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, bool) [with TM = std::complex<double>; TV_ROW = std::complex<double>; TV_COL = std::complex<double>]':
2023-03-01T22:16:09.2297257Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:988:18:   required from here
2023-03-01T22:16:09.2298382Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:70:35: error: 'WIDTH' is not a member of 'ngbla::mat_traits<std::complex<double> >'
2023-03-01T22:16:09.2299273Z    70 |         if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
2023-03-01T22:16:09.2299724Z       |                                   ^~~~~
2023-03-01T22:16:09.2379259Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:70:65: error: 'HEIGHT' is not a member of 'ngbla::mat_traits<std::complex<double> >'
2023-03-01T22:16:09.2379812Z    70 |         if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
2023-03-01T22:16:09.2380183Z       |                                                                 ^~~~~~
2023-03-01T22:16:09.2380854Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:78:33: error: 'HEIGHT' is not a member of 'ngbla::mat_traits<std::complex<double> >'
2023-03-01T22:16:09.2381313Z    78 |     entrysize = mat_traits<TM>::HEIGHT;
2023-03-01T22:16:09.2381609Z       |                                 ^~~~~~
2023-03-01T22:16:09.2382279Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:79:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<std::complex<double> >'
2023-03-01T22:16:09.2382756Z    79 |     iscomplex = mat_traits<TM>::IS_COMPLEX;
2023-03-01T22:16:09.2383065Z       |                                 ^~~~~~~~~~
2023-03-01T22:16:09.2934684Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::MumpsInverse<TM, TV_ROW, TV_COL>::MumpsInverse(const ngla::SparseMatrix<TM, TV_ROW, TV_COL>&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, bool) [with TM = double; TV_ROW = std::complex<double>; TV_COL = std::complex<double>]':
2023-03-01T22:16:09.2935870Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:989:18:   required from here
2023-03-01T22:16:09.2937643Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:70:35: error: 'WIDTH' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.2938220Z    70 |         if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
2023-03-01T22:16:09.2938585Z       |                                   ^~~~~
2023-03-01T22:16:09.2939230Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:70:65: error: 'HEIGHT' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.2939867Z    70 |         if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
2023-03-01T22:16:09.2940224Z       |                                                                 ^~~~~~
2023-03-01T22:16:09.2940858Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:78:33: error: 'HEIGHT' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.2941292Z    78 |     entrysize = mat_traits<TM>::HEIGHT;
2023-03-01T22:16:09.2941573Z       |                                 ^~~~~~
2023-03-01T22:16:09.2952718Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:79:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.2953226Z    79 |     iscomplex = mat_traits<TM>::IS_COMPLEX;
2023-03-01T22:16:09.2953537Z       |                                 ^~~~~~~~~~
2023-03-01T22:16:09.3233126Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::MumpsInverse<TM, TV_ROW, TV_COL>::MumpsInverse(const ngla::SparseMatrix<TM, TV_ROW, TV_COL>&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, bool) [with TM = ngbla::Mat<1, 1, double>; TV_ROW = ngbla::Vec<1, double>; TV_COL = ngbla::Vec<1, double>]':
2023-03-01T22:16:09.3234704Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:991:18:   required from here
2023-03-01T22:16:09.3235905Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:79:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<ngbla::Mat<1, 1, double> >'
2023-03-01T22:16:09.3967059Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::MumpsInverse<TM, TV_ROW, TV_COL>::MumpsInverse(const ngla::SparseMatrix<TM, TV_ROW, TV_COL>&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, bool) [with TM = ngbla::Mat<1, 1, std::complex<double> >; TV_ROW = ngbla::Vec<1, std::complex<double> >; TV_COL = ngbla::Vec<1, std::complex<double> >]':
2023-03-01T22:16:09.3968613Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:992:18:   required from here
2023-03-01T22:16:09.3969793Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:79:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<ngbla::Mat<1, 1, std::complex<double> > >'
2023-03-01T22:16:09.4633531Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::MumpsInverse<TM, TV_ROW, TV_COL>::MumpsInverse(const ngla::SparseMatrix<TM, TV_ROW, TV_COL>&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, bool) [with TM = ngbla::Mat<2, 2, double>; TV_ROW = ngbla::Vec<2, double>; TV_COL = ngbla::Vec<2, double>]':
2023-03-01T22:16:09.4635030Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:995:18:   required from here
2023-03-01T22:16:09.4636150Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:79:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<ngbla::Mat<2, 2, double> >'
2023-03-01T22:16:09.5384538Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::MumpsInverse<TM, TV_ROW, TV_COL>::MumpsInverse(const ngla::SparseMatrix<TM, TV_ROW, TV_COL>&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, bool) [with TM = ngbla::Mat<2, 2, std::complex<double> >; TV_ROW = ngbla::Vec<2, std::complex<double> >; TV_COL = ngbla::Vec<2, std::complex<double> >]':
2023-03-01T22:16:09.5385573Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:996:18:   required from here
2023-03-01T22:16:09.5386339Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:79:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<ngbla::Mat<2, 2, std::complex<double> > >'
2023-03-01T22:16:09.5807744Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::MumpsInverse<TM, TV_ROW, TV_COL>::MumpsInverse(const ngla::SparseMatrix<TM, TV_ROW, TV_COL>&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, bool) [with TM = ngbla::Mat<3, 3, double>; TV_ROW = ngbla::Vec<3, double>; TV_COL = ngbla::Vec<3, double>]':
2023-03-01T22:16:09.5808739Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:999:18:   required from here
2023-03-01T22:16:09.5809498Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:79:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<ngbla::Mat<3, 3, double> >'
2023-03-01T22:16:09.6085504Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::MumpsInverse<TM, TV_ROW, TV_COL>::MumpsInverse(const ngla::SparseMatrix<TM, TV_ROW, TV_COL>&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, bool) [with TM = ngbla::Mat<3, 3, std::complex<double> >; TV_ROW = ngbla::Vec<3, std::complex<double> >; TV_COL = ngbla::Vec<3, std::complex<double> >]':
2023-03-01T22:16:09.6086457Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:1000:18:   required from here
2023-03-01T22:16:09.6087496Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:79:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<ngbla::Mat<3, 3, std::complex<double> > >'
2023-03-01T22:16:09.6438105Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::ParallelMumpsInverse<TM, TVEC>::ParallelMumpsInverse(const ngla::BaseSparseMatrix&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, std::shared_ptr<ngla::ParallelDofs>, bool) [with TM = double; TVEC = double]':
2023-03-01T22:16:09.6439012Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:1031:18:   required from here
2023-03-01T22:16:09.6439845Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:464:35: error: 'WIDTH' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.6440500Z   464 |         if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
2023-03-01T22:16:09.6440835Z       |                                   ^~~~~
2023-03-01T22:16:09.6441450Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:464:65: error: 'HEIGHT' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.6441924Z   464 |         if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
2023-03-01T22:16:09.6442263Z       |                                                                 ^~~~~~
2023-03-01T22:16:09.6453001Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:544:33: error: 'HEIGHT' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.6453473Z   544 |     entrysize = mat_traits<TM>::HEIGHT;
2023-03-01T22:16:09.6453987Z       |                                 ^~~~~~
2023-03-01T22:16:09.6454775Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:545:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.6455452Z   545 |     iscomplex = mat_traits<TM>::IS_COMPLEX;
2023-03-01T22:16:09.6455864Z       |                                 ^~~~~~~~~~
2023-03-01T22:16:09.6722741Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::ParallelMumpsInverse<TM, TVEC>::ParallelMumpsInverse(const ngla::BaseSparseMatrix&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, std::shared_ptr<ngla::ParallelDofs>, bool) [with TM = std::complex<double>; TVEC = std::complex<double>]':
2023-03-01T22:16:09.6724504Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:1032:18:   required from here
2023-03-01T22:16:09.6725272Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:464:35: error: 'WIDTH' is not a member of 'ngbla::mat_traits<std::complex<double> >'
2023-03-01T22:16:09.6725769Z   464 |         if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
2023-03-01T22:16:09.6726103Z       |                                   ^~~~~
2023-03-01T22:16:09.6726745Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:464:65: error: 'HEIGHT' is not a member of 'ngbla::mat_traits<std::complex<double> >'
2023-03-01T22:16:09.6727242Z   464 |         if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
2023-03-01T22:16:09.6727598Z       |                                                                 ^~~~~~
2023-03-01T22:16:09.6741548Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:544:33: error: 'HEIGHT' is not a member of 'ngbla::mat_traits<std::complex<double> >'
2023-03-01T22:16:09.6742082Z   544 |     entrysize = mat_traits<TM>::HEIGHT;
2023-03-01T22:16:09.6742385Z       |                                 ^~~~~~
2023-03-01T22:16:09.6743050Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:545:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<std::complex<double> >'
2023-03-01T22:16:09.6743821Z   545 |     iscomplex = mat_traits<TM>::IS_COMPLEX;
2023-03-01T22:16:09.6744299Z       |                                 ^~~~~~~~~~
2023-03-01T22:16:09.7012839Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::ParallelMumpsInverse<TM, TVEC>::ParallelMumpsInverse(const ngla::BaseSparseMatrix&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, std::shared_ptr<ngla::ParallelDofs>, bool) [with TM = double; TVEC = std::complex<double>]':
2023-03-01T22:16:09.7013983Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:1033:18:   required from here
2023-03-01T22:16:09.7014675Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:464:35: error: 'WIDTH' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.7015137Z   464 |         if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
2023-03-01T22:16:09.7015464Z       |                                   ^~~~~
2023-03-01T22:16:09.7016062Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:464:65: error: 'HEIGHT' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.7016503Z   464 |         if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
2023-03-01T22:16:09.7016853Z       |                                                                 ^~~~~~
2023-03-01T22:16:09.7024741Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:544:33: error: 'HEIGHT' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.7025209Z   544 |     entrysize = mat_traits<TM>::HEIGHT;
2023-03-01T22:16:09.7025505Z       |                                 ^~~~~~
2023-03-01T22:16:09.7026315Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:545:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<double>'
2023-03-01T22:16:09.7026750Z   545 |     iscomplex = mat_traits<TM>::IS_COMPLEX;
2023-03-01T22:16:09.7027029Z       |                                 ^~~~~~~~~~
2023-03-01T22:16:09.7177492Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::ParallelMumpsInverse<TM, TVEC>::ParallelMumpsInverse(const ngla::BaseSparseMatrix&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, std::shared_ptr<ngla::ParallelDofs>, bool) [with TM = ngbla::Mat<1, 1, double>; TVEC = ngbla::Vec<1, double>]':
2023-03-01T22:16:09.7178481Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:1035:18:   required from here
2023-03-01T22:16:09.7179246Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:545:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<ngbla::Mat<1, 1, double> >'
2023-03-01T22:16:09.7508401Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::ParallelMumpsInverse<TM, TVEC>::ParallelMumpsInverse(const ngla::BaseSparseMatrix&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, std::shared_ptr<ngla::ParallelDofs>, bool) [with TM = ngbla::Mat<1, 1, std::complex<double> >; TVEC = ngbla::Vec<1, std::complex<double> >]':
2023-03-01T22:16:09.7509412Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:1036:18:   required from here
2023-03-01T22:16:09.7511049Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:545:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<ngbla::Mat<1, 1, std::complex<double> > >'
2023-03-01T22:16:09.7903571Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::ParallelMumpsInverse<TM, TVEC>::ParallelMumpsInverse(const ngla::BaseSparseMatrix&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, std::shared_ptr<ngla::ParallelDofs>, bool) [with TM = ngbla::Mat<2, 2, double>; TVEC = ngbla::Vec<2, double>]':
2023-03-01T22:16:09.7904972Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:1039:18:   required from here
2023-03-01T22:16:09.7905774Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:545:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<ngbla::Mat<2, 2, double> >'
2023-03-01T22:16:09.8242641Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::ParallelMumpsInverse<TM, TVEC>::ParallelMumpsInverse(const ngla::BaseSparseMatrix&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, std::shared_ptr<ngla::ParallelDofs>, bool) [with TM = ngbla::Mat<2, 2, std::complex<double> >; TVEC = ngbla::Vec<2, std::complex<double> >]':
2023-03-01T22:16:09.8243947Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:1040:18:   required from here
2023-03-01T22:16:09.8244738Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:545:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<ngbla::Mat<2, 2, std::complex<double> > >'
2023-03-01T22:16:09.8607667Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::ParallelMumpsInverse<TM, TVEC>::ParallelMumpsInverse(const ngla::BaseSparseMatrix&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, std::shared_ptr<ngla::ParallelDofs>, bool) [with TM = ngbla::Mat<3, 3, double>; TVEC = ngbla::Vec<3, double>]':
2023-03-01T22:16:09.8609075Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:1043:18:   required from here
2023-03-01T22:16:09.8610145Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:545:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<ngbla::Mat<3, 3, double> >'
2023-03-01T22:16:09.8924183Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp: In instantiation of 'ngla::ParallelMumpsInverse<TM, TVEC>::ParallelMumpsInverse(const ngla::BaseSparseMatrix&, std::shared_ptr<ngcore::BitArray>, std::shared_ptr<const ngcore::Array<int> >, std::shared_ptr<ngla::ParallelDofs>, bool) [with TM = ngbla::Mat<3, 3, std::complex<double> >; TVEC = ngbla::Vec<3, std::complex<double> >]':
2023-03-01T22:16:09.8925191Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:1044:18:   required from here
2023-03-01T22:16:09.8926000Z /tmp/ngsolve-src/linalg/mumpsinverse.cpp:545:33: error: 'IS_COMPLEX' is not a member of 'ngbla::mat_traits<ngbla::Mat<3, 3, std::complex<double> > >'
```

Thanks,
Francesco

--- a/linalg/mumpsinverse.cpp
+++ b/linalg/mumpsinverse.cpp
@@ -67,7 +67,7 @@ namespace ngla
 	    throw Exception("Invalid parameters inner/cluster. Thrown by MumpsInverse.");
 	  }
 	
-	if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
+	if ( int( ngbla::Width<TM>()) != int(ngbla::Height<TM>()) )
 	  {
 	    cout << "Mumps: Each entry in the square matrix has to be a square matrix!" << endl;
 	    throw Exception("No Square Matrix. Thrown by MumpsInverse.");
@@ -75,8 +75,8 @@ namespace ngla
       }
 
 
-    entrysize = mat_traits<TM>::HEIGHT; 
-    iscomplex = mat_traits<TM>::IS_COMPLEX;
+    entrysize = ngbla::Height<TM>();
+    iscomplex = ngbla::IsComplex<TM>();
 
 
     int * colstart = 0;
@@ -461,7 +461,7 @@ namespace ngla
 	    throw Exception("Invalid parameters inner/cluster. Thrown by ParallelMumpsInverse.");
 	  }
 	
-	if ( int( mat_traits<TM>::WIDTH) != int(mat_traits<TM>::HEIGHT) )
+	if ( int(ngbla::Width<TM>()) != int(ngbla::Height<TM>()) )
 	  {
 	    cout << "Mumps: Each entry in the square matrix has to be a square matrix!" << endl;
 	    throw Exception("No Square Matrix. Thrown by ParallelMumpsInverse.");
@@ -541,8 +541,8 @@ namespace ngla
     */
 
 
-    entrysize = mat_traits<TM>::HEIGHT; 
-    iscomplex = mat_traits<TM>::IS_COMPLEX;
+    entrysize = ngbla::Height<TM>();
+    iscomplex = ngbla::IsComplex<TM>();
 
     Array<int> col_indices;
     Array<int> row_indices;
@@ -1121,6 +1121,4 @@ namespace ngla
 
 
 #endif
-
-
 

